src/main/java/org/geysermc/discordbot/commands/PingCommand.java

```diff
@@ -87,7 +90,12 @@ protected void execute(CommandEvent event) {
             return;
         }
 
-        event.getMessage().replyEmbeds(handle(args.get(0))).queue();
+        String ip = args.get(0);
+        if (args.size() > 1) {
+            ip += ":" + args.get(1);
+        }
+
+        event.getMessage().replyEmbeds(handle(ip)).queue();
     }
 
     private MessageEmbed handle(String ip) {
```

Comment:
should we update this method to take ip + port, instead of putting them together and then splitting it.

passing a null Integer is probably fine design wise

Reply:
Yeah this is just the lazy solution - not against using a nullable Integer; I'll update it

Reply:
https://github.com/GeyserMC/GeyserDiscordBot/pull/299/commits/4fa4f638615ba3f1859ff518f3d521c4722df839 fixed here